src/actions/authentication.js

```diff
@@ -23,7 +25,10 @@ export const signIn = async (payload) => {
     await Api.signIn(payload);
     notifier.success({ code: SIGN_IN_SUCCESS });
   } catch (error) {
-    notifier.error({ code: SIGN_IN_ERROR, error });
+    if(error.message === 'Not Found')
+      notifier.error({ code: SIGN_IN_INVALID, error });
+    else
+      notifier.error({ code: SIGN_IN_ERROR, error });
```

Comment:
Looks like you need to setup `Prettier` in your code editor. We are currently using `{}` for `if` and `else` conditions, even if they are one liners. Also, need a space between `if` and `(`.